### PR TITLE
Adding is_final flag in StreamingTranslateSpeechToSpeechResponse 

### PR DESCRIPTION
**Pull request by mohnishparmar**
Adds  an eos flag in the message to mark the last chunk of the stream

### CODE REVIEW

File: riva/proto/riva_tts.proto [hunk 1]
```diff
@@ -65,6 +65,7 @@ message SynthesizeSpeechResponseMetadata {
     string text = 1;
     string processed_text = 2;
     repeated float predicted_durations = 8;
+    bool eos = 9;
```
Comment (virajkarandikar) [inline riva/proto/riva_tts.proto]: this could be `is_final` as we have it for ASR ? Also add a comment.

Comment (mohnishparmar) [reply riva/proto/riva_tts.proto]: done

**Comment by junkin** [comment]
This is implicit in the API  already, when SynthesizeOnline.read() returns false the stream is finished no?

**Comment by mohnishparmar** [comment]
moving the is_final flag as a part nmt response and remove tts proto changes.

File: riva/proto/riva_nmt.proto [hunk 2]
```diff
@@ -76,8 +76,12 @@ message SynthesizeSpeechConfig {
 /*
 *
 */
+
 message StreamingTranslateSpeechToSpeechResponse {
+    oneof streaming_response {
         nvidia.riva.tts.SynthesizeSpeechResponse speech = 1; //from riva_tts.proto
+        bool is_final=2; //end of tts stream
+    }
```
Comment (virajkarandikar) [inline riva/proto/riva_nmt.proto:85]: Fix spacing.

Comment (virajkarandikar) [reply riva/proto/riva_nmt.proto:85]: As I understand `is_final` will be set after every translated sentence. So might be good to add a comment about it.

**Comment by virajkarandikar** [comment]
update MR title.

File: riva/proto/riva_nmt.proto (hunk 2 above)
Comment (rmittal-github) [reply riva/proto/riva_nmt.proto:85]: agree, we should add a comment for the StreamingTranslateSpeechToSpeechResponse message itself and mention one line detail for this flag as well.

Comment (mohnishparmar) [reply riva/proto/riva_nmt.proto:85]: done.

Comment (messiaen) [reply riva/proto/riva_nmt.proto:85]: Based on the comment should we call it `end_of_sentence`?  We use `is_final` in ASR to indicate the transcript will no longer change, which doesn't really apply here.

**Pull request closed without merging**